modules/installation-extend-edge-nodes-aws-local-zones.adoc

```diff
@@ -0,0 +1,114 @@
+
+The `openshift-installer` creates the machine set manifests file with `node-role.kubernetes.io/edge` and `node-role.kubernetes.io/worker` labels applied to each edge worker node that is located in a Local Zone subnet.
+
+.Prerequisites
+
+* You have an access to the OpenShift CLI (`oc`).
+* You deployed your cluster in a Virtual Private Cloud (VPC) with defined Local Zone subnets.
+* You ensured that the machine set for the edge workers on Local Zone subnets specifies the taints for `node-role.kubernetes.io/edge`.
+
+.Procedure
+
+. Create a `deployment` resource YAML file for an example application to be deployed in the edge worker node that operates in a Local Zone subnet. Ensure that you specify the correct tolerations that match the taints for the edge worker node.
++
+.Example of a configured `deployment` resource for an edge worker node that operates in a Local Zone subnet
+[source,yaml]
+----
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: <pvc_name>
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 10Gi
+  storageClassName: gp2-csi <1>
+  volumeMode: Filesystem
+...
+apiVersion: apps/v1
+kind: Deployment <2>
+metadata:
+  name: <local_zone_application> <3>
+  namespace: <local_zone_application_namespace> <4>
+spec:
+  selector: 
+    matchLabels:
+      app: <local_zone_application>
+  replicas: 1
+  template:
+    metadata:
+      labels: 
+        app: <local_zone_application>
+        machine.openshift.io/zone-group: ${ZONE_GROUP_NAME} <5>
+    spec:
+      nodeSelector: <6>
+        machine.openshift.io/zone-group: ${ZONE_GROUP_NAME}
+      tolerations: <7>
+      - key: "node-role.kubernetes.io/edge"
+        operator: "Equal"
+        value: ""
+        effect: "NoSchedule"
+      containers:
+        - image: openshift/origin-node
+          command:
+           - "/bin/socat"
+          args:
+            - TCP4-LISTEN:8080,reuseaddr,fork
+            - EXEC:'/bin/bash -c \"printf \\\"HTTP/1.0 200 OK\r\n\r\n\\\"; sed -e \\\"/^\r/q\\\"\"'
+          imagePullPolicy: Always
+          name: echoserver
+          ports:
+            - containerPort: 8080
+          volumeMounts:
+            - mountPath: "/mnt/storage"
+              name: data
+    volumes:
+    - name: data 
+      persistentVolumeClaim:
+        claimName: <pvc_name>
+----
+<1> `storageClassName`: For the Local Zone configuration, you must specify `gp2-csi`.
+<2> `kind`: Defines the `deployment` resource.
+<3> `name`: Specifies the name of your Local Zone application. For example, `local-zone-demo-app-nyc-1`.
+<4> `namespace:` Defines the namespace for the AWS Local Zone where you want to run the user workload. For example: `local-zone-app-nyc-1a`.
+<5> `machine.openshift.io/zone-group`: Defines the group to where a zone belongs. For example, `us-east-1-nyc-1`.
+<6> `nodeSelector`: Targets edge worker nodes that match the specified labels.
```

Comment:
Updated, but I cannot determine what example would suit the remaining `machine.openshift.io/zone-group` array. I see in the following docs a user-replaced value if <infrastructure_ID) was set for this array. Should I set a similar value, because `{ZONE_GROUP_NAME}` is stated twice in the example.

https://docs.openshift.com/container-platform/4.8/machine_management/creating-infrastructure-machinesets.html#machineset-creating_creating-infrastructure-machinesets